src/reducers/onboarding.js

```diff
@@ -15,6 +15,8 @@ type Step = {
   },
 }
 
+export type DeviceType = 'nanoX' | 'nanoS' | 'blue' | ''
```

Comment:
is it possible we don't have to treat the `''` case? I imagine we could init the value at `nanoS` ? or is it going to actually select it in the UI?

Reply:
it will select it in the UI?